easybuild/easyconfigs/l/libobjcryst/libobjcryst-2017.2.2-intel-2020a.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'SCons'
+
+name = 'libobjcryst'
+version = '2017.2.2'
```

Comment:
@zarybnicky There's apparently also a `2017.2.3` version tagged, any reason we're not using this (that's probably my bad...)

Reply:
It seems 2017.2.2 is the lsat one with a release tarball ("Downloads"), 2017.2.3 is just a Git tag.